sdks/java/extensions/sql/datacatalog/build.gradle

```diff
@@ -0,0 +1,59 @@
+import groovy.json.JsonOutput
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.module' }
+
+applyJavaNature()
+
+dependencies {
+  provided project(path: ":beam-sdks-java-extensions-sql", configuration: "shadow")
+  shadow library.java.grpc_all
```

Comment:
Can we repackage grpc libraies as we have seen leaked grpc that caused class conflict?

Reply:
Or make it provided to remove it from shadowed jar.

Reply:
i'll try

Reply:
relocation wasn't as trivial as I expected, made it provided for now, will try to figure out the relocation later if needed

Reply:
Sounds good